src/ga4gh/vrs/extras/translator.py

```diff
@@ -110,10 +123,15 @@ def translate_from(
         msg = f"Unable to parse data as {', '.join(formats)}"
         raise ValueError(msg)
 
-    def translate_to(self, vo: models._VariationBase, fmt: str) -> str:
-        """Translate vrs object `vo` to named format `fmt`"""
+    def translate_to(self, vo: models._VariationBase, fmt: str, **kwargs) -> str:
```

Comment:
I think our return type is wrong

```suggestion
    def translate_to(self, vo: models._VariationBase, fmt: str, **kwargs) -> list[str]:
```